### ISSUE TITLE
More Spring 2019

### ISSUE THREAD

**Issue by purposelycryptic** [ScudLee/anime-lists#272]
```xml
  <anime anidbid="14103" tvdbid="360553" defaulttvdbseason="1" episodeoffset="" tmdbid="87473" imdbid="">
    <name>Yatogame-chan Kansatsu Nikki</name>
  </anime>
```

**Comment by purposelycryptic** [comment]
```xml
  <anime anidbid="14256" tvdbid="360580" defaulttvdbseason="" episodeoffset="" tmdbid="87474" imdbid="tt9883660">
    <name>Namu Amida Butsu! Utena</name>
  </anime>
```

```xml
  <anime anidbid="14548" tvdbid="352679" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
    <name>Bakumatsu: Ren'ai Bakumatsu Kareshi Gaiden - Crisis</name>
  </anime>
```

```xml
  <anime anidbid="14196" tvdbid="283935" defaulttvdbseason="0" episodeoffset="" tmdbid="570269" imdbid="tt9826484">
    <name>Gekijouban Trinity Seven: Heavens Library to Crimson Lord</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0">;1-3;</mapping>
    </mapping-list>
  </anime>
```

**Comment by purposelycryptic** [comment]
Included in [Pull Request #276 Mappings for Winter/Spring 2019 Anime](https://github.com/ScudLee/anime-lists/issues/276)
